js/validation-hash-tags.js

```diff
@@ -0,0 +1,40 @@
+import { hashtagsInput } from './form.js';
+
+const AMOUNT_HASH_TAGS = 5;
+
+// запись хештегов в массив
+const createHashTags = () => hashtagsInput.value.split(' ');
+
+
+//Проверка написания хештегов по регулярному выражению
+const matchingPattern = () => {
+  const hashtagTemplate = /^#[A-Za-zА-Яа-я0-9]{1,19}$/;
+  return createHashTags().every((elem) => hashtagTemplate.test(elem));
+};
+
+// Проверка на дублирование
+const isDuplicate = () => {
+  const lowerCaseHashTags = createHashTags().map((elem) => elem.toLowerCase());
+  lowerCaseHashTags.sort();
+  const duplicates = lowerCaseHashTags.filter((item, index) =>
+    item === lowerCaseHashTags[index + 1]);
+  return duplicates.length >= 1;
```

Comment:
```suggestion
  return duplicates.length > 0;
```
как думаешь так сработает?

Reply:
Да)